radicale/config.py

```diff
@@ -218,6 +218,11 @@ def _convert_to_bool(value):
             "value": "warning",
             "help": "threshold for the logger",
             "type": logging_level}),
+        ("logfile", {
+            "value": "/tmp/radicale.log",
```

Comment:
I am **not** part of the Radicale team, so this is a quick review by a random developer.

The string "/tmp/radicale.log" is considered True in Python.  As such this will by default enable logging to "/tmp/radicale.log".  As "/tmp" is a world-writable directory, if any other users exist on the system they can cause Radicale to overwrite any file writable by Radicale.  In other words, I'm pretty sure this is a security hole.

Reply:
Logging to `/tmp` in an example is ugly.